core/templates/domain/skill/SkillMasteryObjectFactorySpec.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Change the current year.

Reply:
Done. Also changed the year in `core/templates/domain/skill/SkillMasteryObjectFactory.ts`